CONTRIBUTING.standards.md

```diff
@@ -0,0 +1,110 @@
+# [Project Name] Charter
```

Comment:
Could you please use `{project name}` instead of `[Project Name]` ?

Reply:
Done 👍